insteon-scene-explorer/ise_pack.yaml

```diff
@@ -0,0 +1,289 @@
+#################################
+## Insteon Scene Explorer Configuration
+#################################
+ise_pack:
+  input_select:
+    #################################
+    ## input select used in the GUI. The options are populated in automations
+    #################################
+    insteon_devices:
+      name: ISE - Target Insteon Device
+      options:
+        - Loading (0)
+      icon: mdi:home
+
+    #################################
+    ## input select used in the GUI. The options are populated in automations
+    #################################
+    insteon_modem_groups:
+      name: ISE - Insteon Modem Scenes
+      options:
+        - 25
+
+  input_number:
+    #################################
+    ## input number used in the GUI.
+    #################################
+    target_insteon_group:
+      name: ISE - Target Insteon Scene
+      min: 25
+      max: 253
+      step: 1
+
+  sensor:
+    #################################
+    ## sensor.insteon_groups
+    ##
+    ## This command line template uses "jq" to process the insteon_devices.json into organized insteon scenes.
+    ## It *only* reads the scenes in the hub (not device to device scenes).
+    ## It returns JSON as attributes for only records that are "in_use" in the ALDB.
+    #################################
+    - platform: command_line
+      name: insteon_groups
+      unique_id: "1656717568"
+      scan_interval: 30
+      command: >
+        jq '
+        {
+          groups: [.[0].address as $modemaddress | .[] | .address as $device_address | select(.address != $modemaddress) | .aldb | .[] | .target = $modemaddress | {
+            group: .group,
+            in_use: .in_use,
+            device_address: $device_address,
+            device_addr: ($device_address[0:2] + "." + $device_address[2:-2] + "." + $device_address[-2:]),
+            device_fqn: ($device_address[0:2] + "." + $device_address[2:-2] + "." + $device_address[-2:] + "_" + (.data3|tostring)),
```

Comment:
I added these two items here to reduce the "computations" that were distributed and consolidate them here.  Not sure if all three device_ properties are needed in the future.